[2025-10-20T23:36:16.111+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-10-20T23:36:16.134+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: Airflow_Lab1_TaskFlow_FileIO.scale_data_task scheduled__2025-10-19T00:00:00+00:00 [queued]>
[2025-10-20T23:36:16.142+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: Airflow_Lab1_TaskFlow_FileIO.scale_data_task scheduled__2025-10-19T00:00:00+00:00 [queued]>
[2025-10-20T23:36:16.142+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 2
[2025-10-20T23:36:16.161+0000] {taskinstance.py:2330} INFO - Executing <Task(_PythonDecoratedOperator): scale_data_task> on 2025-10-19 00:00:00+00:00
[2025-10-20T23:36:16.173+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=160) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-10-20T23:36:16.177+0000] {standard_task_runner.py:63} INFO - Started process 173 to run task
[2025-10-20T23:36:16.175+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'Airflow_Lab1_TaskFlow_FileIO', 'scale_data_task', 'scheduled__2025-10-19T00:00:00+00:00', '--job-id', '6', '--raw', '--subdir', 'DAGS_FOLDER/***.py', '--cfg-path', '/tmp/tmpv52hvacl']
[2025-10-20T23:36:16.181+0000] {standard_task_runner.py:91} INFO - Job 6: Subtask scale_data_task
[2025-10-20T23:36:16.260+0000] {task_command.py:426} INFO - Running <TaskInstance: Airflow_Lab1_TaskFlow_FileIO.scale_data_task scheduled__2025-10-19T00:00:00+00:00 [running]> on host 0074cd054b76
[2025-10-20T23:36:16.369+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='your_name' AIRFLOW_CTX_DAG_ID='Airflow_Lab1_TaskFlow_FileIO' AIRFLOW_CTX_TASK_ID='scale_data_task' AIRFLOW_CTX_EXECUTION_DATE='2025-10-19T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-10-19T00:00:00+00:00'
[2025-10-20T23:36:16.370+0000] {taskinstance.py:430} INFO - ::endgroup::
[2025-10-20T23:36:16.372+0000] {airflow.py:70} INFO - Scaling cleaned data at /opt/***/dags/data/cleaned.pkl
[2025-10-20T23:36:16.413+0000] {python.py:237} INFO - Done. Returned value was: /opt/***/dags/data/scaled.npy
[2025-10-20T23:36:16.414+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-10-20T23:36:16.448+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=Airflow_Lab1_TaskFlow_FileIO, task_id=scale_data_task, run_id=scheduled__2025-10-19T00:00:00+00:00, execution_date=20251019T000000, start_date=20251020T233616, end_date=20251020T233616
[2025-10-20T23:36:16.514+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2025-10-20T23:36:16.546+0000] {taskinstance.py:3503} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-10-20T23:36:16.551+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
